src/handler/ts6.rs

```diff
@@ -21,9 +21,10 @@ fn mode_args(
             'f' | 'j' | 'l' if !remove => true,
             _ => false,
         };
-        let arg = match has_arg {
-            true => Some(args.next().unwrap()),
-            false => None,
+        let arg = if has_arg {
+            Some(args.next().unwrap())
+        } else {
+            None
         };
```

Comment:
This sort of pattern can be written quite tersely with [`bool::then`](https://doc.rust-lang.org/std/primitive.bool.html#method.then). The lint for this is `clippy::if_then_some_else_none`.